### ISSUE TITLE
Add Jazzy Documentation

### ISSUE THREAD

**Issue by croossin** [intuit/CardParts#58]
As we are growing, documentation will be a lot to handle - we should use [Jazzy](https://github.com/realm/jazzy) as a way to have clean and concise documentation.

**Comment by xtopolis** [comment]
Are you looking to split the ReadMe into more consumable chunks, or move documentation to markup above methods and classes in the swift files? Or both?

**Comment by xtopolis** [comment]
Here's what it looks like with a minimal amount of Jazzy config: https://right-notebook.surge.sh/

**Comment by croossin** [comment]
@xtopolis Yes that is looking good - so it would just be getting to 100% documentation coverage.

**Comment by xtopolis** [comment]
# Note any Hacktoberfest'ers - if you want to help me on this PR, feel free! #176

**Comment by mohinee** [comment]
@croossin , @xtopolis I am interested in helping you out with this

**Comment by xtopolis** [comment]
@mohinee Awesome, do you need guidance, or do you just want to jump in and add to the PR?

**Comment by mohinee** [comment]
@xtopolis - I might need a little guidance as I have never used jazzy, do let me know what needs to be done so I can at least start

**Comment by xtopolis** [comment]
Invited you to my fork, I'll tag you in the PR with instructions

**Comment by LMVogel** [comment]
I'd like to help, too

**Comment by xtopolis** [comment]
Invited you to the fork @LMVogel

**Comment by xtopolis** [comment]
Updated the PR to make it easier to see how to get started: https://github.com/intuit/CardParts/pull/176
